README.md

```diff
@@ -178,6 +163,29 @@ then allowing dynamic injection of `<script type="importmap-shim">` to immediate
 
 This follows the [dynamic import map specification approach outlined in import map extensions](https://github.com/guybedford/import-maps-extensions).
 
+### Skip Processing
+
+> Stability: Non-spec feature
+
+When loading modules that you know will only use baseline modules features, it is possible to set a rule to explicitly
+opt-out modules from rewriting. This improves performance because those modules then do not need to be processed or transformed at all, so that only local application code is handled and not library code.
+
+This can be configured by setting the `skip` URL regular expression on a `esmsInitOptions` global before loading es-module-shims:
+
+```js
+self.esmsInitOptions = {
```

Comment:
To make this clearer can we instead use `globalThis` or `window`. Self might be confusing to some users.

Reply:
It would also be nice to have a full example of how to load both es-module-shims and set the options together like a user would in practice.

Reply:
Alternatively even a dedicated "init options" section to discuss this mechanic explicitly.

Reply:
I've added the Init Options section here: https://github.com/guybedford/es-module-shims/pull/109/commits/8b3d26ddc7771610454be0ad8d2951129b531dde

And added a small blurb to describe the undocumented onerror hook.

Let me know if there's anything else you'd like to see.